duckduckgo/Android
# Feature/fire button visited sites fix

Task/Issue URL: https://app.asana.com/0/72649045549333/749739069164807
Tech Design URL: 
CC: 

**Description**:
Workaround for the `WebView` bug where visited links were rendered a different colour, even after the history was cleared.

`WebView` bug report: https://bugs.chromium.org/p/chromium/issues/detail?id=709627

**Steps to test this PR**:
1. Click on a link, then navigate back. You should see the link has changed colour
1. Hit the 🔥button, and confirm you want to clear data
1. Verify the 🔥animation displays briefly
1. Verify you are shown the Browser again
1. Visit the same page you were on previously; verify the link colour is normal and not the visited link colour

Repeat the steps above, but with some tougher scenarios for the FireActivity; rotate it, hit the home and return to the app etc...


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
